app/assets/stylesheets/layout.scss

```diff
@@ -54,14 +54,6 @@ aside {
 
   .two-column {
     @include clearfix;
-    .column-left {
-      width: 47%;
-      float: left;
-    }
-    .column-right {
-      width: 47%;
-      float: right;
-    }
```

Comment:
I don't think we can drop these CSS bits. They're used in other pages (e.g., check out the `/downloads` page before and after this change).